app/css/pages/dashboard.css

```diff
@@ -66,7 +66,7 @@
         }
     }
     & header.dashboard-header {
-        @apply flex items-center pt-32 mb-48;
+        @apply flex lg:flex-row flex-col gap-48 items-center pt-32 mb-48;
```

Comment:
This causes the page to render differently.

Compare these:
<img width="1440" alt="Screenshot 2022-10-19 at 11 57 16" src="https://user-images.githubusercontent.com/286476/196673090-be6e6c00-2c96-4286-8f54-01163339185b.png">
 
<img width="1440" alt="Screenshot 2022-10-19 at 11 57 12" src="https://user-images.githubusercontent.com/286476/196673071-45e5bf40-2fce-430d-9d54-78fdceb32960.png">

I think what you want is to either remove the gap at the higher viewport, or just do `gap-y`

```suggestion
        @apply flex lg:flex-row flex-col gap-y-48 items-center pt-32 mb-48;
```